.github/workflows/aks/aks-manual-shutdown.yaml

```diff
@@ -45,4 +45,4 @@ jobs:
           tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
           allow-no-subscriptions: true
       - name: AKS Manual Stop
-        run: ./scripts/aks-manual-shutdown.sh
+        run: ../scripts/aks-manual-shutdown.sh
```

Comment:
is that matching the new setup?

Reply:
@narinder17 looks like the file path is is referencing auto-shutdown.sh when it should be referencing aks-auto-shutdown.sh

Reply:
placing the files in a sub folders, amending the path for workflows and also scripts in the repo for aks.